src/components/LinkCompany/blocks/Companies.tsx

```diff
@@ -0,0 +1,63 @@
+import { Checkbox } from "@deskpro/deskpro-ui";
+import { LoadingSpinner, HorizontalDivider } from "@deskpro/app-sdk";
+import { Card, NoFound } from "@/components/common";
+import { CompanyItem } from "@/components/CompanyItem";
+import { Fragment, type FC } from "react";
+import type { Maybe } from "@/types";
+import type { Account, Company } from "@/services/copper/types";
+
+export type Props = {
+  isLoading: boolean;
+  companies: Company[];
+  account: Maybe<Account>;
+  selectedCompanies: Company[];
+  onChangeSelectedCompanies: (company: Company) => void;
+};
+
+const Companies: FC<Props> = ({
+  account,
+  isLoading,
+  companies,
+  selectedCompanies,
+  onChangeSelectedCompanies,
+}) => {
+  if (isLoading) {
+    return (
+      <LoadingSpinner/>
+    );
+  }
+
+  return (
+    <>
+      {!companies.length
+        ? <NoFound text="No companies found"/>
+        : companies.map((company) => (
+          <Fragment key={company.id}>
+            <Card>
+              <Card.Media>
+                <Checkbox
+                  size={12}
+                  containerStyle={{ marginTop: 4 }}
+                  onChange={() => onChangeSelectedCompanies(company)}
+                  checked={selectedCompanies.some((selectedCompany) => {
+                    return company.id === selectedCompany.id;
+                  })}
+                />
+              </Card.Media>
+              <Card.Body>
+                <CompanyItem
+                  account={account}
+                  company={company}
+                  onClickTitle={() => onChangeSelectedCompanies(company)}
+                />
+              </Card.Body>
+            </Card>
+            <HorizontalDivider/>
```

Comment:
A bit of a guess, but I'm thinking this shouldn't show for the last element in the list?

Reply:
This is to separate list items from each other. Without it, readability is worse.

Reply:
I only meant for the last element. At the moment it is;

```
header
=====
item 1
----
item 2
----
item n
----
```

When it should be
```
header
=====
item 1
----
item 2
----
item n
```
(note that the last divider is gone)

Reply:
in Figma, we have a divider after the last element.

Reply:
Can you provide a link/source please? I've not seen it before and it'd be nice to have a glance at.